Dockerfile

```diff
@@ -25,21 +25,21 @@ RUN go mod download
 ADD ./cmd/ $APP_ROOT/src/cmd/
 ADD ./pkg/ $APP_ROOT/src/pkg/
 
-RUN go build ./cmd/mirroring
-RUN CGO_ENABLED=0 go build -gcflags "all=-N -l"  -o mirroring_debug ./cmd/mirroring
+RUN go build ./cmd/verifier
```

Comment:
as you changing this file can we update the go here as well? if is 1.19 we want then set to 1.19.10 otherwise 1.20.5

Reply:
Updated to 1.20.5